.eslintrc.js

```diff
@@ -7,45 +7,54 @@ const noCyclicModulesImportsConfig = `
 `
 
 module.exports = {
-  extends: [
-    'plugin:eslint-comments/recommended',
-  ],
-  plugins: [
-    'plentific'
-  ],
+  extends: ["plugin:eslint-comments/recommended"],
+  plugins: ["plentific"],
 
   rules: {
-    'eslint-comments/disable-enable-pair': ['error', { allowWholeFile: true }],
-    'eslint-comments/no-unused-disable': 'error',
-    'plentific/debug': 'error',
-    'plentific/no-concurrent-async': 'error',
-    'plentific/no-cyclic-modules-imports': ['error', { config: noCyclicModulesImportsConfig }],
-    'plentific/no-trailing-slash': 'error',
-    'plentific/no-window-location-replace': 'error',
+    "eslint-comments/disable-enable-pair": ["error", { allowWholeFile: true }],
+    "eslint-comments/no-unused-disable": "error",
+    "plentific/debug": "error",
+    "plentific/no-concurrent-async": "error",
+    "plentific/no-deprecated-components": [
+      "error",
+      {
+        "examples/app/components/ComponentDeprecated":
+          "examples/app/components/Component",
```

Comment:
Leaving it as suggestion here, potentially this can be extended to an array, where as a second optional value we can provide a link to storybook where detailed examples can help developers with migrations.

Reply:
I was thinking about that as well but the value is quite universal you can even do 

```json
{
  "examples/app/components/ComponentDeprecated":
    "examples/app/components/Component (see documentation on https://bla.com)",
}
```

Reply:
but probably i'd won't do exact matching because if deprecated component will be moved, and somebody forget to adjust the rule, it will stop matching, maybe so if we do matching like import ends with, you can still do exact match or just sub matching `components/ComponentDeprecated` or just `ComponentDeprecated`

Reply:
yeah, as @rostislav-simonik-plc mentioned - i did similar in the `dashboard` PR already: https://github.com/plentific/dashboard/pull/25141/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R32

Reply:
i wouldn't lock it into Storybook, because not every substitution is UX/UI/Storybook/Blueprint-related